library/include/rocfft.h

```diff
@@ -182,10 +183,10 @@ ROCFFT_EXPORT rocfft_status rocfft_plan_create(rocfft_plan*                  pla
  *  @param[in] info execution info handle created by
  * rocfft_execution_info_create
  *  */
-ROCFFT_EXPORT rocfft_status rocfft_execute(const rocfft_plan     plan,
-                                           void*                 in_buffer[],
-                                           void*                 out_buffer[],
-                                           rocfft_execution_info info);
+ROCFFT_EXPORT rocfft_status rocfft_execute(const rocfft_plan           plan,
+                                           void*                       in_buffer[],
+                                           void*                       out_buffer[],
+                                           const rocfft_execution_info info);
```

Comment:
info cannot be const, I think it was intended to act as both an input & output parameter, the library may be returning information back to user

Reply:
Sounds good; done.